table.csv

```diff
@@ -1,20 +1,25 @@
 codec,              description,              code
 
 miscellaneous,,
-bin,                raw binary,               0x55
+bin,                raw binary,               0x55 # DUPLICATE CONFLICT
 
 bases encodings,,
-base1,              unary,                    0x01
-base2,              binary (0 and 1),         0x
-base8,              octal,                    0x07
-base10,             decimal,                  0x09
-base16,             hexadecimal,              0x
-base32,             rfc4648,                  0x
-base32hex,          rfc4648,                  0x
-base58flickr,       base58 flicker,           0x
-base58btc,          base58 bitcoin,           0x
-base64,             rfc4648,                  0x
-base64url,          rfc4648,                  0x
+identity,           raw binary,               0x00 ("\0") # DUPLICATE
```

Comment:
We need to clarify the distinction between this and `bin` and if there even is one. If not, we may want to drop this one and make everything use `bin`. That would also affect the identity hash function still under discussion.